tests/test_codex_feed_hooks.py

```
@@ -1373,6 +1373,68 @@ def test_install_recovers_hook_trust_when_cmux_marker_is_unclosed(
+        )
+
+    config_toml = config_path.read_text(encoding="utf-8")
+    if '[plugins."documents@openai-primary-runtime"]' not in config_toml:
+        raise AssertionError(f"documents plugin table was removed: {config_toml!r}")
+    if '[plugins."browser@openai-bundled"]' not in config_toml:
+        raise AssertionError(f"browser plugin table was removed: {config_toml!r}")
+    if 'trusted_hash = "sha256:stale"' in config_toml:
+        raise AssertionError(f"stale cmux hook trust was preserved: {config_toml!r}")
+    trust_begin = "# cmux-codex-hook-trust-f5cc24da-7a09-4b20-a756-89e7786f6738 begin"
+    if config_toml.count(trust_begin) != 1:
+        raise AssertionError(f"install did not write one fresh cmux hook trust marker: {config_toml!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)